include/CppUTest/Utest.h

```diff
@@ -171,6 +171,10 @@ class UtestShell
 
     virtual SimpleString getMacroName() const;
     TestResult *getTestResult();
+
+    static const NormalTestTerminator normalTestTerminator_;
+    static const CrashingTestTerminator crashingTestTerminator_;
```

Comment:
Why this change?

Reply:
I moved them from statics in the source file to protected statics in the class to make the testing easier.

Reply:
Yeah, please don't do that ;) I've commented below how to change the test so that this isn't needed.